ggshield/iac/iac_scan_models.py

```diff
@@ -33,40 +43,86 @@ class IaCDiffScanResult(Base):
     )
 
 
+# TODO: move this schema into pygitguardian
 IaCDiffScanResultSchema = marshmallow_dataclass.class_schema(
     IaCDiffScanResult, BaseSchema
 )
 
+logger = logging.getLogger(__name__)
+
+
+# TODO: Delete this after moving iac_diff_scan into pygitguardian
+def is_ok(resp: Response) -> bool:
+    """
+    is_ok returns True is the API responded with 200
+    and the content type is JSON.
+    """
+    return (
+        resp.headers["content-type"] == "application/json" and resp.status_code == 200
+    )
+
+
+# TODO: Delete this after moving iac_diff_scan into pygitguardian
+def load_detail(resp: Response) -> Detail:
+    """
+    load_detail loads a Detail from a response
+    be it JSON or html.
+
+    :param resp: API response
+    :type resp: Response
+    :return: detail object of response
+    :rtype: Detail
+    """
+    if resp.headers["content-type"] == "application/json":
+        data = resp.json()
+    else:
+        data = {"detail": resp.text}
 
-class MockClient(GGClient):
-    # TODO: move this into GGClient
-    def mock_api_iac_diff_scan(
+    return cast(Detail, Detail.SCHEMA.load(data))
```

Comment:
`is_ok()`, `load_detail()` and `DEFAULT_API_VERSION` do not have to be duplicated. They do the same thing as the py-gitguardian code.

Reply:
True, thank you